File: app/src/main/java/reschikov/geekbrains/androidAdvancedLevel/weatherApplication/data/opencage/Status.java
package reschikov.geekbrains.androidadvancedlevel.weatherapplication.data.opencage;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Status {

    @SerializedName("code")
    @Expose
    private Integer code;
    @SerializedName("message")
    @Expose
    private String message;

    public Integer getCode() {
        return code;
    }

    public String getMessage() {
        return message;
    }
}

File: app/src/main/java/reschikov/geekbrains/androidadvancedlevel/weatherapplication/data/network/model/data/opencage/Status.kt
package reschikov.geekbrains.androidadvancedlevel.weatherapplication.data.network.model.data.opencage

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Status(@SerializedName("code") @Expose val code: Int,
                  @SerializedName("message") @Expose val message: String)
